Automattic/abacus
# Filter out test data

**This PR filters out test data.**
- Filters out experiments and metrics starting with `explat_test_`

Resolves #320 




## How has this been tested?






- Automated tests that cover added/changed functionality
- Manual testing with production data (locally):
  - Visit `/experiments` with and without `?debug=true`, notice the filtering of `explat_test_`
  - Visit `/metrics` with and without `?debug=true`, notice the filtering of `explat_test_`
  - Visit `/experiments/new` with and without `?debug=true`, notice the filtering of `explat_test_` of metric options
  - Visit `/experiments/20015/wizard-edit` with and without `?debug=true`, notice the filtering of `explat_test_`
